charts/event-handler/README.md

```diff
@@ -0,0 +1,222 @@
+
+Once the files `client.key` and `client.crt` were created successfully, the following command can be used to create a new secret (`ca.crt` is also included since we'll need it to verify we are connecting to the right fluentd):
+
+```
+kubectl create secret generic teleport-plugin-event-handler-client-tls --from-file="ca.crt=ca.crt,client.key=client.key,client.crt=client.crt"
+```
+
+### Installing the plugin
+
+```
+helm repo add teleport https://charts.releases.teleport.dev/
+```
+
+```shell
+helm install teleport-plugin-event-handler teleport/teleport-plugin-event-handler --values teleport-plugin-event-handler-values.yaml
+```
+
+Example `teleport-plugin-event-handler-values.yaml`:
+
+```yaml
+teleport:
+  address: teleport.example.com:443
+  identitySecretName: teleport-plugin-event-handler-identity
+
+eventHandler:
+  storagePath: "/var/lib/teleport/plugins/event-handler/storage"
+  timeout: "10s"
+  batch: 20
+  namespace: "default"
+
+fluentd:
+  url: ""
+  sessionUrl: ""
+  certificate:
+    secretName: "teleport-plugin-event-handler-client-tls"
+    caPath: "ca.crt"
+    certPath: "client.crt"
+    keyPath: "client.key"
+```
+
+See [Settings](#settings) for more details.
+
+## Settings
+
+The following values can be set for the Helm chart:
+
+<table>
+  <tr>
+    <th>Name</th>
+    <th>Description</th>
+    <th>Type</th>
+    <th>Default</th>
+    <th>Required</th>
+  </tr>
+
+  <tr>
+    <td><code>teleport.address</code></td>
+    <td>Host/port combination of the teleport auth server</td>
+    <td>string</td>
+    <td><code>""</code></td>
+    <td>yes</td>
+  </tr>
+  <tr>
+    <td><code>teleport.identitySecretName</code></td>
+    <td>Name of the Kubernetes secret that contains the credentials for the connection</td>
+    <td>string</td>
+    <td><code>""</code></td>
+    <td>yes</td>
+  </tr>
+  <tr>
+    <td><code>teleport.identitySecretPath</code></td>
+    <td>Key of the field in the secret specified by <code>teleport.identitySecretName</code></td>
+    <td>string</td>
+    <td><code>"auth_id"</code></td>
+    <td>no</td>
+  </tr>
+
+  <tr>
+    <td><code>eventHandler.storagePath</code></td>
+    <td>Path to the directory where `event-handler`'s state is stored</td>
+    <td>string</td>
+    <td><code>"/var/lib/teleport/plugins/event-handler/storage"</code></td>
+    <td>no</td>
+  </tr>
+  <tr>
+    <td><code>eventHandler.timeout</code></td>
+    <td>Maximum time to wait for incoming events before sending them to fluentd.</td>
+    <td>string</td>
+    <td><code>"10s"</code></td>
+    <td>no</td>
+  </tr>
+  <tr>
+    <td><code>eventHandler.batch</code></td>
+    <td>Maximum number of events fetched from Teleport in one request</td>
+    <td>string</td>
+    <td><code>20</code></td>
+    <td>no</td>
+  </tr>
+  <tr>
+    <td><code>eventHandler.namespace</code></td>
+    <td>Namespace where the events </td>
```

Comment:
This seems like a fragment. Perhaps "... where the events are stored?"

Reply:
It's the Teleport namespace where the events are coming in, fixed it! Thanks!